app/jinja_filters.py

```diff
@@ -52,6 +52,29 @@ def format_percentage(value):
     return '{}%'.format(value)
 
 
+@blueprint.app_template_filter()
+def format_address_list(user_entered_address=None, metadata_address=None):
+    """
+        This function format_address_list accepts two addresses, a user submitted address and a metadata address
+        If all the items in address list are 'Undefined' (nothing in answer_store) then we retrieve the metadata.
+
+        :param user_entered_address user entered address
+        :param metadata_address is metadata address
+        :return: the value of the address to be piped
+    """
+    if all(isinstance(a, Undefined) for a in user_entered_address):
```

Comment:
Sorry to be super picky, but could we rename `a` to `field` or something like that?  Makes it easier at a glance that it's a list of fields